packages/gatsby/package.json

```
@@ -120,7 +120,7 @@
     "url-loader": "^1.0.1",
     "uuid": "^3.1.0",
     "v8-compile-cache": "^1.1.0",
-    "webpack": "^4.12.0",
+    "webpack": "~4.28.2",
```

The linked issue suggests `4.28.4`? https://github.com/webpack/webpack/issues/8656#issuecomment-456153381

```suggestion
    "webpack": "~4.28.4",
```

Is this not accurate?

---

Ah yes, I think I snagged from 4.28.2 from the top post. Since this is using a tilde match, the specific patch version doesn’t matter, though may as well use the latest as of today (which is 4.28.4). You can see in the lockfile that’s what’s being pulled in.

---

The lockfile is what _we_ pull in for our monorepo, not really what would get exposed to developers using gatsby! But yeah, `~4.28.2` should resolve to `4.28.4` so I think we're fine here.

---

Yea, I understand.

I feel like pinning the patch version is short-sighted. The tilde match seeeems like it’ll be okay.